src/app/public/modules/split-view/split-view.component.html

```diff
@@ -0,0 +1,70 @@
+<div
+  class="sky-split-view"
+  [@blockAnimationOnLoad]
+>
+  <div *ngIf="isDrawerVisible"
+    class="sky-split-view-drawer"
+    role="region"
+    [attr.aria-label]="drawerComponent?.ariaLabel"
+    [id]="'split-view-drawer-' + splitViewId"
+    [style.width.px]="drawerWidth"
+    [@drawerEnter]
+  >
+    <ng-content
+      select="sky-split-view-drawer"
+    ></ng-content>
+  </div>
+
+  <input *ngIf="isDrawerVisible && !isMobile"
+    class="sky-split-view-resize-handle"
+    role="separator"
+    step="10"
+    type="range"
+    [attr.aria-controls]="'split-view-drawer-' + splitViewId"
+    [attr.aria-label]="'skyux_split_view_resize_handle' | skyLibResources"
+    [attr.aria-valuemax]="drawerWidthMax"
+    [attr.aria-valuemin]="drawerWidthMin"
+    [attr.aria-valuenow]="drawerWidth"
+    [max]="drawerWidthMax"
+    [min]="drawerWidthMin"
+    [style.left.px]="drawerWidth - 7"
+    [value]="drawerWidth"
+    (change)="onResizeHandleChange($event)"
+    (mousedown)="onMouseDown($event)"
+  />
+
+  <div *ngIf="workspaceVisible"
+    class="sky-split-view-workspace"
+    role="region"
+    [attr.aria-label]="workspaceComponent?.ariaLabel"
+    [@workspaceEnter]
+    (@workspaceEnter.done)="onWorkspaceEnterComplete($event)"
+  >
+
+    <div *ngIf="!isDrawerVisible"
```

Comment:
Using `ngIf` removes the element from the DOM. Does this affect the accessibility of the drawer-workspace relationship? If so, maybe you could use `[hidden]` instead?

Reply:
That's a really good point. Hadn't thought of that. It might have some impact on those animations, because they rely on the element being destroyed. Let me look into it...

Reply:
Cool. FYI, this question relates to the `beforeWorkspaceClose` question, above.

Reply:
Done. Now using `[hidden]`.